app/workers/send_eoc_deadline_reminder_email_to_candidates_worker.rb

```diff
@@ -6,7 +6,7 @@ class SendEocDeadlineReminderEmailToCandidatesWorker
   def perform
     return unless CycleTimetable.need_to_send_deadline_reminder?
 
-    GroupedRelationBatchDelivery.new(
+    BatchDelivery.new(
```

Comment:
I remember @tomas-stefano mentioning that there's a bug in one of these. Something around calling `.size` rather than `.count` or something like that 🤔

Reply:
Yeah, it was fixed a couple of cards ago, but I realised when reviewing this specific task that I had used the wrong job for this relationship. So just fixing that here. I'll do some manual testing with it using production data (not actually sending emails, just getting the data) to make sure it works as expected before the email goes out.